src/stactools/global_mangrove_watch/stac.py

```diff
@@ -1,68 +1,309 @@
+respectively. Sources of error included a mis-registration in the SAR mosaic datasets, 
+which could only be partially corrected for, but also confusion in fragmented areas of 
+mangroves, such as around aquaculture ponds. Overall, 152,604 km2 (133,996 - 176,910) 
+of mangroves were identified for 1996, with this decreasing by -5,245 km2 
+(-13,587 - 3686) resulting in a total extent of 147,359 km2 (127,925 - 168,895) in 
+2020, and representing an estimated loss of 3.4 % over the 24-year time period. The 
+Global Mangrove Watch Version 3.0 represents the most comprehensive record of global 
+mangrove change achieved to date and is expected to support a wide range of activities, 
+including the ongoing monitoring of the global coastal environment, defining and 
+assessments of progress towards conservation targets, protected area planning and risk 
+assessments of mangrove ecosystems worldwide.
+
+The paper which goes along with this dataset is available at the following reference:
+
+{PUBLICATION_CITATION}"""
+
+
+COG_ASSET_NAME = "cog"
+CHANGE_ASSET_NAME = "change_cog"
 
-import stactools.core.create
+ITEM_ASSETS = {
+    COG_ASSET_NAME: ItemAssetDefinition.create(
+        title="Mangrove cover",
+        description="Gridded estimate of mangrove cover",
+        media_type=MediaType.COG.value,
+        roles=["data"],
+    ),
+    CHANGE_ASSET_NAME: ItemAssetDefinition.create(
+        title="Mangrove cover change since 1996",
+        description="Gridded estimate of mangrove cover change since 1996",
+        media_type=MediaType.COG.value,
+        roles=["data"],
+    ),
+}
 
 
 def create_collection() -> Collection:
     """Creates a STAC Collection.
 
-    This function should create a collection for this dataset. See `the STAC
-    specification
-    <https://github.com/radiantearth/stac-spec/blob/master/collection-spec/collection-spec.md>`_
-    for information about collection fields, and
-    `Collection<https://pystac.readthedocs.io/en/latest/api.html#collection>`_
-    for information about the PySTAC class.
-
     Returns:
         Collection: STAC Collection object
     """
     extent = Extent(
         SpatialExtent([[-180.0, 90.0, 180.0, -90.0]]),
-        TemporalExtent([[datetime.now(tz=timezone.utc), None]]),
+        TemporalExtent(
+            [
+                [
+                    datetime(1996, 1, 1, tzinfo=timezone.utc),
+                    datetime(2020, 12, 31, 23, 59, 59, tzinfo=timezone.utc),
+                ]
+            ]
+        ),
     )
 
     collection = Collection(
-        id="example-collection",
-        title="Example collection",
-        description="An example collection",
+        id=COLLECTION_ID,
+        title=TITLE,
+        description=format_multiline_string(DESCRIPTION),
+        keywords=KEYWORDS,
         extent=extent,
-        extra_fields={"custom_attribute": "foo"},
+        license="CC-BY-4.0",
+        providers=[
```

Comment:
Where does one inject the new HOST if you're copying the data (and unzipping it)?

Reply:
We could do that in the `stactools-pipeline` code. I will do the download/unzip/upload operation there, then in the script that creates the `collection` object we can add (or replace) the `HOST` provider.